config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-release-1-2-upgrades.yaml

```diff
@@ -304,7 +304,7 @@ periodics:
   spec:
     serviceAccountName: prowjob-default-sa
     containers:
-      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20221221-461b6105cf-1.24
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20221223-736a4da5ba-1.24
```

Comment:
The image used here should be the same as other tests: `gcr.io/k8s-staging-test-infra/kubekins-e2e:v20221221-461b6105cf-1.24`.